docs/install-1.7.md

```diff
@@ -21,65 +32,75 @@ kubernetes infrastructure CAs or certificates. This script should be
 `source`ed to define all of the variables it contains in the current
 shell process.
 
-The aggregator is a new feature of kubernetes and is an alpha API
-running as a separate pod in kubernetes v1.6. The aggregator is
-enabled by default in kubernetes v1.7 as a beta API, and is the
-default secure endpoint directly integrated into the core kubernetes
-APIServer.
+For background on why we need to deal with certificates and CA's at all, you 
+may read the [the auth doc](auth.md) (this is not required, however).
 
-For background on why we're messing with certificates and CA's at all,
-please check [the auth doc](auth.md).
+## Install `cfssl` Tools
 
-# Steps
+Before we continue, you'll need to install the `cfssl` tools:
 
-## Prerequisites
-You need `cfssl` tools, install with `go get -u github.com/cloudflare/cfssl/cmd/...`.
+```console
```

Comment:
not 100% sure how much this affects the output, but consistently annotating the verbatim sections. Can take another pass through if this is desired.

Reply:
Not sure what you want here?